plaidml/bridge/openvino/plaidml_builder.cpp

```diff
@@ -238,4 +281,11 @@ plaidml::Program buildProgram(const std::shared_ptr<const ngraph::Function>& fun
   return ProgramBuilder(func, netName, inputsInfo, outputsInfo).build();
 }
 
+plaidml::Program buildNodeProgram(const std::shared_ptr<ngraph::Node>& node) {
+  static std::once_flag once;
```

Comment:
It's ok to call `call_once` multiple times, and each should probably have its own flags.

Reply:
Thanks for correcting me on this point

Reply:
OK, I think I see your concern now, yes I agree we should have both `buildProgram` and `buildNodeProgram` use a single registration that is gated by shared flags (so that one or the other does the registration and only once).

Reply:
I am not 100% sure about call_once functionality but declaration of static once_flag is moved outside of both buildProgram and buildNodeProgram function, that should make registerOps() called only once. Let me know if this looks good.